packages/driver/src/cypress/command_queue.ts

```diff
@@ -541,6 +541,7 @@ export class CommandQueue extends Queue<$Command> {
         current.fail()
       }
 
+      Cypress.action('cy:command:failed', current, err)
```

Comment:
double checking - Should this be command:fails OR command:skipped dep on the command state of queued vs pending?

this would be primarily observers around cy.session spans and error handling if I remember corretly.

Reply:
I don't expect skipped to count as an error. There is an explicit `skipped:command:end` message called. Even so, the span would be ended with the state logged, but it would also include the error.